lib/fork.js

```diff
@@ -13,7 +13,8 @@ module.exports = function (file, opts) {
 		tty: process.stdout.isTTY ? {
 			columns: process.stdout.columns,
 			rows: process.stdout.rows
-		} : false
+		} : false,
+		projectRoot: process.cwd()
```

Comment:
This isn't necessarily the project root. Perhaps `parentWorkingDirectory`? Bit verbose though.